src/components/Template/Template.stories.mdx

```diff
@@ -0,0 +1,410 @@
+            <TemplateContentSidebar>
+              <Placeholder>Right Sidebar</Placeholder>
+            </TemplateContentSidebar>
+          )}
+        </TemplateContent>
+        <TemplateFooter>
+          <Placeholder modifiers={["short"]}>Footer</Placeholder>
+        </TemplateFooter>
+      </Template>
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Template with Props" />
+
+## Template With Sidebar
+
+In order to render a sidebar, two steps must be taken.
+
+1. The `TemplateContent` component needs to be aware of it. Pass the location
+   of the sidebar to the `sidebar` prop as either "left" or "right".
+
+```jsx
+<TemplateContent sidebar="left">
+```
+
+2. Render `TemplateContentSidebar` and `TemplateContentPrimary` next to each
+   other in the appropriate order. If the sidebar must show left, then
+   `TemplateContentSidebar` comes before `TemplateContentPrimary`. Otherwise,
+   `TemplateContentPrimary` must come before `TemplateContentSidebar`.
+
+```jsx
+// Render left sidebar
+<TemplateContent sidebar="left">
+  <TemplateContentSidebar>
+    // ...
+  </TemplateContentSidebar>
+  <TemplateContentPrimary>
+    // ...
+  </TemplateContentPrimary>
+</TemplateContent>
+
+// Render right sidebar
+<TemplateContent sidebar="right">
+  <TemplateContentPrimary>
+    // ...
+  </TemplateContentPrimary>
+  <TemplateContentSidebar>
+    // ...
+  </TemplateContentSidebar>
+</TemplateContent>
+```
+
+<Canvas>
+  <Story name="Template with Sidebar">
+    <Template>
+      <TemplateHeader>
+        <Placeholder modifiers={["short"]}>NYPL Header</Placeholder>
+        <TemplateContentBreakout>
+          <Placeholder modifiers={["short"]}>Breadcrumbs</Placeholder>
+          <Placeholder modifiers={["short"]}>Hero</Placeholder>
+        </TemplateContentBreakout>
+      </TemplateHeader>
+      <TemplateContent sidebar="right">
+        <TemplateContentTop>
+          <Placeholder>Content Top</Placeholder>
+        </TemplateContentTop>
+        <TemplateContentPrimary>
+          <Placeholder>Main Content</Placeholder>
+          <Placeholder modifiers={["short"]}>More Content</Placeholder>
+        </TemplateContentPrimary>
+        <TemplateContentSidebar>
+          <Placeholder>Right Sidebar</Placeholder>
+        </TemplateContentSidebar>
+      </TemplateContent>
+      <TemplateFooter>
+        <Placeholder modifiers={["short"]}>Footer</Placeholder>
+      </TemplateFooter>
+    </Template>
+  </Story>
+</Canvas>
+
+## Template Components
+
+The components consist of:
+
+- `Template`
+- `TemplateHeader`
+- `TemplateContentBreakout`
+- `TemplateContent`
+- `TemplateContentTop`
+- `TemplateContentPrimary`
+- `TemplateContentSidebar`
+- `TemplateFooter`
+
+### Template
+
+```
+<Template>
+  // ...
+</Template>
+```
+
+<Description of={Template} />
+
+### TemplateHeader
+
+```
+<Template>
+  <TemplateHeader>
+    // ...
+  </TemplateHeader>
+<Template>
+```
+
+<Description of={TemplateHeader} />
+
+### TemplateContentBreakout
```

Comment:
That's true. In the previous examples, this was only used in the header. So maybe it should be `TemplateHeaderBreakout` since only components there go edge-to-edge.

Reply:
Yes, that works.  The breakout mixin is used in other components, so that is something that should be moved into a global file at some point.